.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "dotnetAcquisitionExtension.existingDotnetPath": [
     "/usr/local/dotnet/current/dotnet"
-  ]
+  ],
+  "editor.fontFamily": "'CaskaydiaCove NF'"
```

Comment:
This should be a preference in your user settings, not workspace settings. I use that font too, but we shouldn't force it on everyone :)

Reply:
Agree, fixed it.